changelog.d/pa-2902.fixed

```diff
@@ -0,0 +1,2 @@
+Rust: Fixed a bug where taint could not propagate through
```

Comment:
You have more context than me for sure, but I'm not sure I'd qualify this as a bug, may it be a feature?

Reply:
honestly, the line is hazy to me too. i'm not really sure where one ends and the other begins... "fixed", because we should have been able to do this in the first place. but also, it's reasonable that we didn't, and we just downscoped it, so feature?? not sure.

Reply:
i can rename it to feature, i don't mind

Reply:
No strong opinions on my side, just wanted to make you think about it.

Reply:
Features are something more positive than bugs, so that if both could apply, perhaps that could be a reason to prefer presenting something as a feature. 🤔 🤷‍♂️

Reply:
oh that's a good point, I didn't think of that one. will change!